media/js/base/nav-main-resp.js

```diff
@@ -414,11 +414,9 @@ NavMain.openSmallMenu = function()
         return;
     }
 
-    $('#nav-main-menu')
-        .slideDown(150)
-        .removeAttr('aria-hidden');
-
-    NavMain.toggleButton.addClass('open').attr('aria-expanded', true);
+    $('#nav-main-menu').slideDown(150, function() {
+        NavMain.toggleButton.addClass('open').attr('aria-expanded', true);
+    }).removeAttr('aria-hidden');
```

Comment:
Ah, yes good suggestion. We don't need it for this test directly, but I think it makes sense to make sure both functions match in terms of timing.